src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexBuilder.cs

```diff
@@ -64,6 +67,7 @@ internal sealed class SymbolicRegexBuilder<TElement> where TElement : notnull
         /// </summary>
         internal DfaMatchingState<TElement>[]? _statearray;
         internal DfaMatchingState<TElement>[]? _delta;
+        internal DfaMatchingState<TElement>[]? _capturingStatearray;
```

Comment:
Nit: it'd be nice if these were _stateArray and _capturingStateArray, but we can clean those up subsequently